src/common/numa.cc

```diff
@@ -137,3 +137,37 @@ int get_numa_node_cpu_set(
   ::close(fd);
   return r;
 }
+
+#elif defined(__FreeBSD__)
+
+int parse_cpu_set_list(const char *s,
+		       size_t *cpu_set_size,
+		       cpu_set_t *cpu_set)
+{
+  return 0;
+}
+
+std::string cpu_set_to_str_list(size_t cpu_set_size,
+				const cpu_set_t *cpu_set)
+{
+  std::string r;
+  r = "0";
+  return r;
+}
+
+std::set<int> cpu_set_to_set(size_t cpu_set_size,
+			     const cpu_set_t *cpu_set)
+{
+  set<int> r;
+  return r;
+}
+
+int get_numa_node_cpu_set(
+  int node,
+  size_t *cpu_set_size,
+  cpu_set_t *cpu_set)
+{
+  return 0;
```

Comment:
might want to return `-ENOTSUP` here.

Reply:
@tchaikov 
Will have to check if returning `-SOMETHING` does not run into errors, warnings or asserts later on.
As long as `ENOTSUP` does not start printing anoying loging about not being supported I'm oke with that.

Reply:
@tchaikov 
I concur, and only 2 pesky messages are listed.
Pushed new version